docs/source/create_channel/channel_policies.md

```diff
@@ -0,0 +1,121 @@
+
+## Channel policies and Access Control Lists
+
+The policies within the channel configuration are also referenced by [Access Control Lists (ACLs)](../access_control.html) that are used to restrict access to Fabric resources used by the channel. The ACLs extend the policies within the channel configuration to govern the **processes** of the channel. You can see the default ACLs in the [sample configtx.yaml file](http://github.com/hyperledger/fabric/blob/{BRANCH}/sampleconfig/configtx.yaml). Each ACL refers to a channel policy using the path. For example, the following ACL restricts who can invoke a chaincode based on the `/Channel/Application/Writers` policy:
+```
+# ACL policy for invoking chaincodes on peer
+peer/Propose: /Channel/Application/Writers
+```
+
+Most of the default ACLs point to the ImplicitMeta policies in the application section of the channel configuration. To extend the example above, an organization can invoke a chaincode if they can satisfy the `/Channel/Application/Writers` policy.
+
+  ![channel writer policies](application-writers.png)  
+
+*Figure 4: The peer/Propose ACL is satisfied by the /Channel/Application/Writers policy. This policy can be satisfied by a transaction submitted by a client application from any organization with the writers signature policy.*
+
+## Orderer policies
+
+The ImplicitMeta policies in the **Orderer** section of `configtx.yaml` govern the ordering nodes of a channel in the a similar way as the **Application** section governs the peer organizations. The ImplicitMeta policies point to the signature policies associated with the organizations that are ordering service administrators.
+
+  ![Orderer policies](orderer-policies.png)  
+
+*Figure 5: The Channel/Orderer/Admins policy points to the Admins signature policies associated with the administrators of the ordering service.*
+
+If you use the default policies, a majority of orderer organizations are required to approve the addition or removal of an ordering node.
+
+  ![Orderer policies](orderer-admins.png)  
+
+*Figure 6: A request submitted to remove an ordering node from the channel contains signatures from the three ordering organizations in the network, satisfying the Channel/Orderer/Admins policy. The Org3 check is in light green because the signature was not required to reach to a majority.*
+
+The `Channel/Orderer/BlockValidation` policy governs which ordering nodes have the ability to write new blocks to the channel. Because all ordering administrators have the writers signature policy, all ordering organizations can write new blocks to the channel.
```

Comment:
This doesn't make sense to me.  I would say:

The `Channel/Orderer/BlockValidation` policy is used by peers when receiving blocks (either from ordering, or via gossip).  The peers check that the signature set attesting to the block's validity satisfies the block validation policy to confirm that the block was generated by ordering.  By default, any ordering org 'Writers' are authorized to attest to a block's validity.

Reply:
An edited version of your suggestion: ```The `Channel/Orderer/BlockValidation` policy is used by peers to validate that new blocks being added to the channel were generated by an ordering node that is part of the channel consenter set, and that the block was not tampered with or created by another peer organization. By default, any orderer organization with a `writers` signature policy can create and validate blocks for the channel.```